packages/react/src/combobox/root/AriaCombobox.tsx

```diff
@@ -1270,17 +1271,16 @@ interface ComboboxRootProps<ItemValue> {
    * The items to be displayed in the list.
    * Can be either a flat array of items or an array of groups with items.
    */
-  items?: readonly ItemValue[] | readonly Group<ItemValue>[];
+  items?: readonly any[] | readonly Group<any>[];
```

Comment:
Could `any` be avoided here? 🤔 
In general, I feel like the current `"ItemValue[] | Group<ItemValue>[] | undefined"` in docs types is slightly more clear than the suggested `"any[] | Group[] | undefined"`. 🙈

Reply:
Ok, makes sense.
But maybe the docs could resolve it more precisely?
Not the `<any>` is excluded from the docs. 🙈